mineral-ui/mineral-ui
# [Dialog] [Flex] Fix layout issues

### Description

* Add `minWidth` prop to FlexItem
  * Add demo example
* Fix additional layout issues
  * Ensure DialogBody & FlexItem children do not extend beyond element boundary
* Update Dialog Composition example to illustrate layout scenario

### Motivation and context

Fix issues reported by users in chat.

### Screenshots, videos, or demo, if appropriate

https://layout-bugs--mineral-ui.netlify.com/components/flex-item/min-width

### How to test


1. `cd mineral-ui && npm start`
1. Navigate to FlexItem > [Min Width example](https://layout-bugs--mineral-ui.netlify.com/components/flex-item/min-width)
1. Confirm that the Select is truncated properly
1. Navigate to Dialog > Composition example (only viewable locally)
1. Confirm that the Select is truncated properly

### Types of changes

- Feature

### Checklist
